mvpa2/tests/test_hyperalignment.py

```diff
@@ -36,11 +36,13 @@ class HyperAlignmentTests(unittest.TestCase):
     @sweepargs(zscore_all=(False, True))
     @sweepargs(zscore_common=(False, True))
     @sweepargs(ref_ds=(None, 2))
+    @sweepargs(level1_equal_weight=(False, True))
     @reseed_rng()
-    def test_basic_functioning(self, ref_ds, zscore_common, zscore_all):
+    def test_basic_functioning(self, ref_ds, zscore_common, zscore_all, level1_equal_weight):
         ha = Hyperalignment(ref_ds=ref_ds,
                             zscore_all=zscore_all,
-                            zscore_common=zscore_common)
+                            zscore_common=zscore_common,
+                            level1_equal_weight=level1_equal_weight)
```

Comment:
that is a start! ;) although too expensive one I must say, but how do we know that the "logic" is actually working? ;-)

@feilong could you please add a simple test for `mean_xy` which would demonstrate that if we provide similar to the ones used in the code weighting, it would produce the desired equivalently balanced contribution? 
I know that it sounds "not that interesting", but that is why they are called "unit-tests".

Reply:
No problem, just added a new commit with the tests for `mean_xy`.